sentry/src/test/java/io/sentry/MainEventProcessorTest.kt

```diff
@@ -342,6 +341,36 @@ class MainEventProcessorTest {
         assertEquals("optionsHost", event.serverName)
     }
 
+    @Test
+    fun `Transaction sets server name`() {
```

Comment:
I don't follow the naming. `server name is set on transaction` and so on I think better reflects whats happening here.

Reply:
indeed, naming could be improved